doc/glm/second_level_model.rst

```diff
@@ -0,0 +1,105 @@
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+
+Design matrix
+=============
+
+As with first level models, a design matrix needs to be defined before fitting a second level model. Some common second level models are one-sample and two-sample t-tests. A quick look into setting up design matrices for these models is provided here.
+
+
+One-sample testing
+------------------
+
+In one-sample testing, activity associated with a predictor variable is tested against some form of baseline activity. Therefore, predictors must be defined in order to set up a one-sample t-test. For example, consider the first level contrast of left-vs-right button press, which can be downloaded using :func:`nilearn.datasets.fetch_localizer_contrasts`::
+
+  from nilearn.datasets import fetch_localizer_contrasts
+  n_subjects = 16
+  data = fetch_localizer_contrasts(["left vs right button press"], n_subjects,
+                               get_tmaps=True)
+
+A first level model has been fit to this data and the contrasts maps for left-vs-right button press activation are available in the `cmaps` parameter. The second level model is fit using these contrast maps thus::
+
+  from nilearn.stats.second_level_model import make_second_level_design_matrix
+  second_level_input = data['cmaps']
+  design_matrix = pd.DataFrame([1] * len(second_level_input), columns=['intercept'])
+
+
+Two-sample testing
+------------------
+
+In two-sample testing, the activity attributed to two predictors is contrasted against each other, and so, two predictors need to be defined to fit the model. Consider the dataset that has activation corresponding to the presentation of horizonal and vertical checkerboards. This dataset can also be downloaded using :func:`nilearn.datasets.fetch_localizer_contrasts`::
+
+  from nilearn.datasets import fetch_localizer_contrasts
+  n_subject = 16
+  sample_vertical = fetch_localizer_contrasts(
+      ["vertical checkerboard"], n_subjects, get_tmaps=True)
+  sample_horizontal = fetch_localizer_contrasts(
+      ["horizontal checkerboard"], n_subjects, get_tmaps=True)
+  second_level_input = sample_vertical['cmaps'] + sample_horizontal['cmaps']
+
+Set up the design matrix::
+
+  import numpy as np
+  condition_effect = np.hstack(([1] * n_subjects, [- 1] * n_subjects))
+  subject_effect = np.vstack((np.eye(n_subjects), np.eye(n_subjects)))
+  subjects = ['S%02d' % i for i in range(1, n_subjects + 1)]
+  design_matrix = pd.DataFrame(
+      np.hstack((condition_effect[:, np.newaxis], subject_effect)),
+      columns=['vertical vs horizontal'] + subjects)
+
+As with first level models, the design matrix can be visualized using :func:`nilearn.reporting.plot_design_matrix()`
+
+.. image:: ../auto_examples/05_glm_second_level_models/images/sphx_glr_plot_second_level_two_sample_test_001.png
+   :target: ../auto_examples/05_glm_second_level_models/plot_second_level_two_sample_test.html
+
```

Comment:
Actually this one is a paired t-test. Should be decribed as such.

Reply:
I agree. The label and description for this example should be changed in the Examples section as well.